src/web/Content/client/dumpling.py

```diff
@@ -944,6 +956,24 @@ def _add_key_if_not_exists(dictProp, key, val):
         if not key in dictProp:
             dictProp[key] = val
 
+    @staticmethod
+    def _create_hang_dump(pid, outpath, debuggerpath):
+        osStr = platform.system().lower()
+        command = ""
+        if osStr == 'linux':
+            command = "./" + debuggerpath + " " + pid + " --name " + outpath + "\memdum" + pid + ".dmp"
+        elif osStr == 'windows':
+            outputpath = outpath + "\memdum" + pid + ".dmp"        
+            command = debuggerpath + " -p "+ pid + " -c " + '".dump /ma '+outputpath+';.detach;q"'
+        else:
+            return
+
+        print "creating dump"
+        try:
+            subprocess.call(command)
+        except OSError as e:
+            print "Not able to create Dump for process " + pid, e        
```

Comment:
Note this may happen if the process has quit since you enumerated the process tree. I imagine it takes time to create the dumps, so this is possible. It looks like you carry on so that's fine.